### PR TITLE
Pin the last remaining github actions

### PR DESCRIPTION
**Pull request by davidkelliott**
As per security guidance from github

### CODE REVIEW

File: .github/workflows/terraform-static-analysis.yml [hunk 1]
```diff
@@ -62,11 +62,11 @@ jobs:
     if: github.event_name == 'schedule'
     steps:
     - name: Checkout
-      uses: actions/checkout@v3
+      uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
       with:
         fetch-depth: 0
     - name: Run Analysis
-      uses: ministryofjustice/github-actions/terraform-static-analysis@main
+      uses: ministryofjustice/github-actions/terraform-static-analysis@9a8388207b7cedd2684d766121108c90cb010234
```
Comment (julialawrence) [inline .github/workflows/terraform-static-analysis.yml:69]: Are we gonna track updates to this action manually?

**Review by julialawrence — COMMENTED** [review]
Are we gonna track versions of static-code-analysis actions manually?

**Comment by davidkelliott** [comment]
> Are we gonna track versions of static-code-analysis actions manually?

I think for now, I need to speak to ops eng to see often they are making new releases.  I checked and there are releases but the latest change that I made to the repo hasn't been incorporated yet.

**Review by julialawrence — APPROVED** [review]
LGTM

**Pull request merged**